<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PduDevWid</class>
 <widget class="QWidget" name="PduDevWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>66</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>电压(V)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>电流(A)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>功率(KW)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="volLcd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="curLcd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="powLcd">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>电能：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="eleLab">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0KWh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>温度：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>湿度：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tmpLab">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="humLab">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Line 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Line 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Line 3</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
